src/main/java/hazards/Enemy.java

```diff
@@ -0,0 +1,46 @@
+package hazards;
+
+/** The abstract class which all enemies inherit from. */
+public abstract class Enemy {
```

Comment:
The Entity class (abstract) will have a similar functionality such as storing the starting positions. We can keep this class as it is for now, but perhaps we can merge it together later on?